bundles/af-core/src/test/resources/form/radiobutton/exporter-radiobutton-customized.json

```diff
@@ -16,8 +16,8 @@
   },
   "enforceEnum": true,
   "enumNames": [
-    "Item 1",
-    "Item 2"
+    "<p>Item 1</p>",
```

Comment:
Can you add a test case where if the options are not rich text, there is <p> tag ?

Reply:
aren't we considering by default that enumNames will be rich text?